### ISSUE TITLE
kine master is pached now 

### ISSUE THREAD

**Issue by RSDMODS** [iSteal-it/script#3]
working with 6.2.1 now got new updates and patched 6.3.0 new version 

now need update to ip address please update

**Comment by urstrulycherry** [comment]
Hey is kinemaster working for you now?

**Comment by RSDMODS** [comment]
yes i used crack kinemaster direct install with help of scarlet ,
sideloadly , esign.

**Comment by urstrulycherry** [comment]
Is it for iPhone?

**Comment by RSDMODS** [comment]
yes bro for iphone just ping me on telegram @RSDMODS

**Comment by RSDMODS** [comment]
old kinemaster working with vpn.
